clients/web/src/views/layout/ProgressListView.js

```diff
@@ -51,6 +54,15 @@ var ProgressListView = View.extend({
         this._onUpdate();
     },
 
+    _clearProgress: function () {
+        _.each(Object.keys(this._map), (key) => {
+            this._map[key].destroy();
+            this._map[key].remove();
+            delete this._map[key];
+            this._onUpdate();
+        });
+    },
+
     _onUpdate: function () {
         if (_.isEmpty(this._map)) {
```

Comment:
This is outside the scope of your changes, but the whole `_onUpdate` function body could be simplified as:
```
this.$el.toggle( !_.isEmpty(this._map) );
```